OpenRA.Utility/Program.cs

```diff
@@ -43,9 +43,16 @@ static void Main(string[] args)
 			Log.AddChannel("perf", null);
 			Log.AddChannel("debug", null);
 
-			Game.InitializeSettings(Arguments.Empty);
+			var arguments = new Arguments(args);
+			args = arguments.GetSkippedValues();
```

Comment:
This skipped value approach doesn't sit well with me, it feels like something that tries/wants to be generic but is only ever going to be useful in this specific case.

IMO it would be a lot cleaner to search for `args.FirstOrDefault(a => a.StartsWith("Engine.ModSearchPaths="))` instead of shoehorning `Arguments` in here.